core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -1181,7 +1181,7 @@ object TestUtils extends Logging {
                        transactionVerificationEnabled: Boolean = false,
                        log: Option[UnifiedLog] = None,
                        remoteStorageSystemEnable: Boolean = false,
-                       initialTaskDelayMs: Long = ServerLogConfigs.LOG_INITIAL_TASK_DELAY_MS_DEFAULT): LogManager = {
+                       initialTaskDelayMs: Long = ServerLogConfigs.LOG_INITIAL_TASK_DELAY_MS_DEFAULT_INTEGRATION_TEST): LogManager = {
```

Comment:
We can change it here for logManager tests. But that's not a place for integration test. The integration test will start up brokers with broker configurations, so that we won't use `TestUtils.createLogManager` method here. Have you checked `IntegrationTestHarness.scala` where we set broker configurations for integration test brokers?

Reply:
Yes, `createBrokerConfigs` is a place to override some configs for testing, like we set `props.put(ServerLogConfigs.LOG_DELETE_DELAY_MS_CONFIG, "1000")`, so that we can quickly delete a log file to speed up the test. Similar things like
```
// Reduce number of threads per broker
    props.put(ServerConfigs.NUM_NETWORK_THREADS_CONFIG, "2")
    props.put(ServerConfigs.BACKGROUND_THREADS_CONFIG, "2")
```
As the comment said, we want to reduce the number of threads per broker, to speed up the tests.
That's why we should add a smaller initialTaskDelay here to speed up tests.
Does that make sense?

Reply:
@showuon got it, completed the changes. Can you review it again. Thanks